charms/k8s/charmcraft.yaml

```diff
@@ -21,3 +59,20 @@ config:
 parts:
   charm:
     build-packages: [git]
+    charm-entrypoint: worker/src/charm.py
+  lib:
+    # move the ./worker/lib path to ./lib since 
+    # charmcraft assumes it to be there once the charm runs
+    after: [charm]
+    plugin: nil
+    source: ./
+    override-prime: |
+      mv worker/lib lib
```

Comment:
So, it seems that charmcraft expects the `lib` folder to always been at the same path level as charmcraft.yaml and other "charm files" when run on the unit.  This moves the lib file to the top level during the charm build process.